test/Project.toml

```diff
@@ -0,0 +1,4 @@
+[deps]
```

Comment:
`test` should not have a `Project.toml`, instead the main `Project.toml` has a section with the `Test` libraries.

Reply:
Here's an example: https://github.com/JuliaSmoothOptimizers/SolverTools.jl/blob/master/Project.toml